astroquery/esa/jwst/core.py

```diff
@@ -1035,6 +1035,40 @@ def get_obs_products(self, *, observation_id=None, cal_level="ALL",
 
         return files
 
+    def download_files_from_program(self, proposal_id, *, product_type=None, verbose=False):
```

Comment:
Ahh, one more nitpick: The same method in `esa.hubble` has the signature:

```
    def download_files_from_program(self, program, *, folder=None, calibration_level=None,
                                    data_product_type=None, intent=None,
                                    obs_collection=None, instrument_name=None,
                                    filters=None, only_fits=False):
                                    ```
                                    
Would you mind making this consistent by using arg names `program` instead of `proposal_id`, and also maybe `data_product_type` instead of `product_type`? Though the latter doesn't seem to be exactly the same, consistency may be useful.

I'll also consider this as a follow-up item for a new PR, so will merge this as currently is.

Reply:
@davidglt - not sure you've seen this as I merged the PR as is, without waiting for an API cleanup discussion.